Mathlib/Topology/Bases.lean

```diff
@@ -1034,3 +1034,9 @@ protected theorem Embedding.secondCountableTopology [SecondCountableTopology β]
     SecondCountableTopology α :=
   hf.1.secondCountableTopology
 #align embedding.second_countable_topology Embedding.secondCountableTopology
+
+protected theorem Embedding.separableSpace {α : Type u_1} {β : Type u_2} [TopologicalSpace α]
```

Comment:
You can reuse the alpha and beta from the variables

Reply:
Indeed.